smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java

```diff
@@ -112,24 +157,517 @@ public void processMessage(Message message) {
             }
         });
 
-        MucCreateConfigFormHandle handle = mucAsSeenByOne.createOrJoin(Resourcepart.from("one-" + randomString));
-        if (handle != null) {
-            handle.makeInstant();
-        }
+        createMUC(mucAsSeenByOne, "one-" + randomString);
         mucAsSeenByTwo.join(Resourcepart.from("two-" + randomString));
-
         mucAsSeenByOne.sendMessage(mucMessage);
-        resultSyncPoint.waitForResult(timeout);
+        try {
+            resultSyncPoint.waitForResult(timeout);
+        } catch (TimeoutException e) {
+            throw new AssertionError("Failed to receive presence");
```

Comment:
Always wrap the causing exception.

Reply:
This does not wrap the causing exception, i.e. `throw new AssertionError("Timeout waiting for result", e)`.

But after looking again at it, why do you catch it anyways? Simply let the method throw the `TimeoutException`. This is already done in the other test methods.